tests/parsers/agilent_gen5/testdata/multi_read_modes/jira_cust_8062_bug.txt

```diff
@@ -0,0 +1,80 @@
+

```

Comment:
Can you please rename this from `jira_cust_8062_bug` to something like `absorbance_and_fluorescence`? We don't want to reference JIRA tickets in allotropy.